src/screens/SendSheet.js

```diff
@@ -300,12 +300,30 @@ export default function SendSheet(props) {
     }
 
     let submitSuccess = false;
+    let updatedGasLimit = null;
+    // Attempt to update gas limit before sending ERC20 / ERC721
+    if (selected?.address !== 'eth') {
```

Comment:
can we use ETH_ADDRESS from `@rainbow-me/references` for `eth`

Reply:
YES!